Documentation/ApiOverview/Seo/_ExampleWebsiteTitle/_WebsiteTitleProvider.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
I don't like this example: Just implement the interface and use `getTitle()` instead of `doSomeThing()`.